src/cpp/session/modules/SessionProfiler.R

```diff
@@ -80,3 +101,11 @@
       .rs.enqueClientEvent("rprof_stopped");
    }
 })
+
+options("profvis.print" = function(x)
```

Comment:
Consider only setting this option if it's currently unset -- realize it's pretty unlikely but if a user sets their own `provis.print` in e.g. `.Rprofile` we don't want to stomp it here.